conf/example.yaml

```diff
@@ -31,9 +31,9 @@ truncate_table: false
 # to map it back to the logical timestamp of the operation that wrote this value.
 clock:
   approximate_monotonic:
-    historySize: 7300
-    epochPeriod: 1
-    epochTimeUnit: "SECONDS"
+    history_size: 7300
+    epoch_length: 1
+    epoch_time_unit: "SECONDS"
```

Comment:
the prop names incorrectly matched the Java var names

Reply:
well spotted! However, if this has happened, we need to rename java props as well, since jackson will attempt to dump properties with their field names.

Reply:
turns out there are more places like this. If you like you can just leave it and i'll fix it later, as you wish.

Reply:
Fixed remaining var names in `DebugApproximateMonotonicClockConfiguration`.